Library/Formula/gpac.rb

```diff
@@ -12,14 +12,11 @@ class Gpac < Formula
   homepage 'http://gpac.wp.mines-telecom.fr/'
 
   stable do
-    url 'https://downloads.sourceforge.net/gpac/gpac-0.5.0.tar.gz'
-    sha1 '48ba16272bfa153abb281ff8ed31b5dddf60cf20'
-
-    # Fixes build against ffmpeg 2.x; backported from upstream SVN
-    patch :DATA
```

Comment:
Sure thing, sorry I didn't see that at the end. I'll try to send out an
update tomorrow morning CST.

On Tue, Feb 17, 2015, 5:15 PM Misty De Meo <notifications@github.com> wrote:

> In Library/Formula/gpac.rb
> <https://github.com/Homebrew/homebrew/pull/36913#discussion_r24864760>:
>
> > @@ -12,14 +12,11 @@ class Gpac < Formula
> >    homepage 'http://gpac.wp.mines-telecom.fr/'
> >
> >    stable do
> > -    url 'https://downloads.sourceforge.net/gpac/gpac-0.5.0.tar.gz'
> > -    sha1 '48ba16272bfa153abb281ff8ed31b5dddf60cf20'
> > -
> > -    # Fixes build against ffmpeg 2.x; backported from upstream SVN
> > -    patch :DATA
>
> If you're removing the patch, can you also delete __END__ and all the
> lines following it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/36913/files#r24864760>.
>